test/WebSites/RazorWebSite/Services/TestViewTagHelperComponent.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace RazorWebSite
+{
+    public class TestViewTagHelperComponent : ITagHelperComponent
```

Comment:
Not quite sure why we've got multiple almost-identical components in this web site. Couldn't the bits that differ e.g. the appended content be passed into the constructor or (hmm, `Order`) set?